backend/Dockerfile.celery

```diff
@@ -0,0 +1,24 @@
+# Base image
+FROM python:3.9-slim
```

Comment:
This is a bit of an older image, would a newer one work?

Reply:
I see it's used in some other dockerfile as well.

Reply:
I'll test with a newer image